frontend/src/pages/PrizePage/index.js

```diff
@@ -1,25 +1,44 @@
 import { useState } from "react";
-import Prize from "components/Prize";
+import { COMPANY } from "constants/Subvisual.js";
+
+import { ReactComponent as Tshirt } from "assets/svgs/prize/t-shirt.svg";
+import { ReactComponent as NFT } from "assets/svgs/prize/nft.svg";
+
+import Button from "components/Button";
+import Text from "components/Typography/Text";
+import Title from "components/Typography/Title";
 
 import styles from "./index.module.css";
 
 const PrizePage = () => {
-  const [prize, setPrize] = useState("tshirt");
+  const prizes = {
+    tshirt: <Tshirt />,
+    nft: <NFT />,
+  };
 
-  const handleClick = () => {
-    if (prize === "tshirt") {
-      setPrize("nft");
-    }
+  const [prize, setPrize] = useState("nft");
 
-    if (prize === "nft") {
-      setPrize("tshirt");
-    }
+  const handleClick = () => {
+    prize === "nft" ? setPrize("tshirt") : setPrize("nft");
   };
 
   return (
-    <div className={styles.center}>
+    <div className={styles.root}>
       <button onClick={handleClick}>Change Prize</button>
-      <Prize name={prize} />
+
+      <div className="center">
+        <Title>{`Congrats, you won a ${prize}`}</Title>
+
+        <div className={styles.item}>{prizes[prize]}</div>
```

Comment:
`item` seems very generic, why not calling it `prize`?

Reply:
We'll fix all of these before merging to main tomorrow morning :smile: